src/topology/continuous_function/compact.lean

```diff
@@ -260,6 +260,17 @@ variables {𝕜 : Type*} {γ : Type*} [normed_field 𝕜] [normed_ring γ] [norm
 instance [nonempty α] : normed_algebra 𝕜 C(α, γ) :=
 { norm_algebra_map_eq := λ c, (norm_algebra_map_eq (α →ᵇ γ) c : _), }
 
+instance has_continuous_smul' : has_continuous_smul 𝕜 C(α, γ) :=
```

Comment:
Thanks for the change!  Is this instance now superseded by the other instance, or is there a case I'm missing where this one applies and the other doesn't?

Reply:
Yes, it is.